sdb/Core/config.py

```diff
@@ -23,8 +23,8 @@ class BasePipelineConfig(BaseModel):
     model: Model = Model(name="anything-v4.5", id="ckpt/anything-v4.5-vae-swapped", local=False)
     width: int = 512
     height: int = 512
-    batch_size: int = 2
-    sample_steps: int = 35
+    batch_size: int = 1
+    num_inference_steps: int = 50
```

Comment:
Since the default schedule is DPM++ interference steps doesnt need to be this high by default to get good looking images